README.md

```diff
@@ -1,67 +1,71 @@
-# Author-Endorser-Wizards
-
-Python program to walk through the author and endorser workflow to create schemas and cred defs 
-
-## Docker Instructions
-
-### Running the Endorser
-once you have cloned the repo,
-
-* docker build -t endorser-wizard -f endorser-docker-file . (the dot is important, as it signifys the current directory)
-* docker run --rm -it -v \<Path to git repository\>/author-endorser-wizards:/root/author-endorser-wizards:z -v \<Path to .indy_client\>/wallet:/root/.indy_client/wallet:z -v \<Path to .indy_client\>/pool:/root/.indy_client/pool:z endorser-wizard 
-  * \(If on windows run: docker run --rm -it -v \<Path to git repository\>\author-endorser-wizards:/root/author-endorser-wizards:z -v \<Path to .indy_client\>\wallet:/root/.indy_client/wallet:z -v \<Path to .indy_client\>\pool:/root/.indy_client/pool:z endorser-wizard\)
-### Running the Author
-* docker build -t author-wizard -f author-docker-file . (the dot is important, as it signifys the current directory)
-* docker run --rm -it -v \<Path to git repository\>/author-endorser-wizards:/root/author-endorser-wizards:z -v \<Path to .indy_client\>/wallet:/root/.indy_client/wallet:z -v \<Path to .indy_client\>/pool:/root/.indy_client/pool:z author-wizard
-  * \(If on windows run: docker run --rm -it -v \<Path to git repository\>\author-endorser-wizards:/root/author-endorser-wizards:z -v \<Path to .indy_client\>\wallet:/root/.indy_client/wallet:z -v \<Path to .indy_client\>\pool:/root/.indy_client/pool:z author-wizard
-
-
-## Alternate Setup Instructions
-If your not using Docker, here are the individual setup and running instructions you can follow to run the author and endorser wizards
-
-### Linux 18.04
-
- On a clean install of Ubuntu 18.04 the following works to install and run the Appropriate Wizard.  Please adjust if appropriate for your environment.
-* sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys CE7709D068DB5E88
-* sudo add-apt-repository "deb https://repo.sovrin.org/sdk/deb bionic stable"
-* sudo apt-get update
-* sudo apt-get upgrade
-* sudo apt-get install -y libindy
-* sudo apt install python3.8 -y
-* sudo update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.6 1
-* sudo update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.8 2
-* sudo apt install python3-pip git
-* pip3 install --upgrade pip
-* pip3 install aiohttp base58 indy Python3-indy asyncio
+[**Docker Instructions**](#docker)  
+[Endorser](#endorser)  
+[Author](#author)  
+[**Alternate Setup Instructions**](#alternate)  
+[Linux](#linux)  
+[Windows](#windows)  
+[Mac](#mac)  
+[**Running Instructions**](#running)
 
+# Author-Endorser-Wizards
+Python program to walk through the author and endorser workflow to create schemas and cred defs.
+
+## Docker Instructions <a id="docker"></a>
```

Comment:
Rename to docker setup instructions?  Newbie coming to the site might be confused as to why there is even a docker setup here.  We should probably add some explanatory steps here or above as a quick explanation for help with navigating?
"The easiest way to get up and running the Author and Endorser wizards is to use the docker setup...(etc.)"  Probably the preceeding paragraph should be expanded to 2-3 more sentences of explanation.

Reply:
I think it should say "Docker Setup Instructions." We can more depth when we get a chance.